<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>map_plot</class>
 <widget class="QWidget" name="map_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="log_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Log Current</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="current_min_label">
       <property name="text">
        <string>Current Min:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="current_min_lineedit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="current_min_unit_label">
       <property name="text">
        <string>[A]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="current_max_label">
       <property name="text">
        <string>Current Max:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="current_max_lineedit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="current_max_unit_label">
       <property name="text">
        <string>[A]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scale_current_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Scale Current</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scale_rf_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Scale RF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scale_dc_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Scale DC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="scanline_checkbox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show Scanline</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="plot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="navigation_bar" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
